### ISSUE TITLE
ES6 spread operator

### ISSUE THREAD

**Issue by renatoi** [js-next/react-style#91]
While I understand the push to use ES6 syntax, I believe it would be better if you could transpile to ES5 the disted version on npm, otherwise it would require developers to transpile react-style inside of 'node_modules' which sounds to me as a bad idea. Unless if you think this is the correct way to do it. Publishing with the ES6 syntax would be ok to me when we have more support for it, specially on the node.js environment (spread operators are still not supported). This problem has been mentioned in Issue #73.

**Comment by trungpham** [comment]
Should we create a new es5 compatible branch?

**Comment by trungpham** [comment]
If anyone is running into issue then you can use my branch.

```
npm install -S trungpham/react-style#es5-compatible
```

```
ERROR in ./~/react-style/lib/ReactElementExtended.js
Module parse failed: /Users/trungpham/projects/edcast-ui/node_modules/react-style/lib/ReactElementExtended.js Line 19: Unexpected token .
You may need an appropriate loader to handle this file type.
| 
| var originalCreateElement = ReactElement.createElement;
| ReactElement.createElement = function(type, props, ...children) {
|   if (props && typeof type === 'string') {
|     props = assign(props, buildProps(props));
 @ ./~/react-style/lib/index.js 3:0-33
```

**Comment by phated** [comment]
:+1: as called out in https://github.com/SanderSpies/react-material/issues/81 - the dependencies need to be transpiled also

**Comment by SanderSpies** [comment]
This is fixed now (0.4.0).

**Comment by trungpham** [comment]
awesome
